docs/tutorials/api/makers.ipynb

```diff
@@ -317,7 +318,22 @@
     "per observation into a larger dataset. During the stacking the safe data\n",
     "range mask (`MapDataset.mask_safe`) is applied by setting data outside to\n",
     "zero, then data is added to the larger map dataset. To stack multiple\n",
-    "observations, the larger dataset must be created first."
+    "observations, the larger dataset must be created first.\n",
+    "\n",
+    "The data reduction loop shown above can be done throught the `DatasetsMaker`.  Its `stack_datasets` option offers the possibily to stack or not the output datasets, and the `n_jobs` option allow to use multiple processes on run:"
```

Comment:
I would add in bold face that the order in which you pass the makers is important

Reply:
added